libbeat/logp/eventlog_windows.go

```diff
@@ -0,0 +1,91 @@
+package logp
+
+import (
+	"strings"
+
+	"github.com/pkg/errors"
+	"go.uber.org/zap/zapcore"
+	"golang.org/x/sys/windows/svc/eventlog"
+)
+
+const (
+	eventID  = 100
+	supports = eventlog.Warning | eventlog.Info | eventlog.Info
```

Comment:
Why does `eventlog.Info` show up twice? Intentional or should it be Debug?

Reply:
That should be `Error`. Will push a fix.